pkg/cli/vex.go

```diff
@@ -38,12 +38,18 @@ more about the VEX tooling powering wolfictk see: https://github.com/chainguard-
 	}
 
 	addPackages(cmd)
-
+	addSBOM(cmd)
 	return cmd
 }
 
+var vexCfg = vex.Config{
+	Distro:     "wolfi",
```

Comment:
(non-blocking for this PR) Maybe we should get rid of the `distro` field here. Now that we're working in `wolfictl`, there shouldn't be functionality for non-Wolfi distros

Reply:
Ok, sounds good. And going a little bit beyond, (thinking about the ?distro=qualifier) maybe we should prepare and design these tools to work in a scenario where we have more than one flavor of Wolfi.